src/pages/view-test-result/components/test-summary-card/test-summary-card.ts

```diff
@@ -31,4 +31,10 @@ export class TestSummaryCardComponent {
 
   getFlattenArray = (data: string[]): string => flattenArray(data);
 
+  /**
+   * Check the value passed in is true/false - used to suppress field in HTML only when undefined.
+   * @param {boolean} data
+   * @return {boolean}
+   */
+  isBoolean = (data: boolean): boolean => typeof data === 'boolean';
```

Comment:
This could be a function called `shouldDisplayLicenceProvided` and the implementation should just use the `isEmpty` function from lodash to define if the license is empty.

Reply:
or the isBoolean function

Reply:
@ammarhaiderbjss Yeah i've updated it to use isBoolean, isEmpty didn't seem to do what i wanted, thanks!